### PR TITLE
491

### PR DESCRIPTION
**Pull request by mcpower**
I don't think I can get it down any further.

### CODE REVIEW

**Comment by mcpower** [comment]
lol jk saved 4 more bytes

**Comment by mcpower** [comment]
5 more bytes.

**Comment by mcpower** [comment]
You could save more bytes by removing `newline`, or you could (slightly) speed it up at the cost of bytes by introducing a similar variable for commas.

I think the current tradeoff (store newline, don't store comma) is a good balance between speed and bytes, but you can definitely go one way or the other.

**Comment by mcpower** [comment]
6 more bytes.

File: index.js [hunk 1]
```diff
@@ -45,59 +46,64 @@ export function *iter(source) {
...
       ++i;
     }
 
-    if (i >= length) {
+    if (!(i < length)) {
       break;
     }
 
-    if (i > newline) {
-      newline = nextIndex('\n');
```
Comment (samthor) [inline index.js:57]: Don't we need this?

Comment (mcpower) [reply index.js:57]: Yes... but it's now in-lined when `newline` is used. You don't actually need to update `newline` every loop, as it's only used in one place.

**Pull request merged**